tests/serverspec/default.yml

```diff
@@ -67,8 +67,8 @@
   roles:
     - ansible-role-virtual-interface
   vars:
-    virtual_interface:
-      gre0:
+    virtual_interfaces:
```

Comment:
i meant the variable name should not have been changed.

I should have developed rules for role names and variables to prevent confusion like this (there are already confusing names, such as `ansible-role-apt-repo` and `ansible-role-freebsd-repos`.